experiment/fix_testgrid_config.py

```diff
@@ -47,6 +47,7 @@
     "sig-release-1.",
     "sig-release-master-upgrade",
     "gke-cos-containerd",
+    "gci-gke",
```

Comment:
google-gke?

Reply:
Uh...I misunderstood the naming. Then google-gke should be sufficient for both `gke-cos-containerd` and `gci-gke`. Updated.